src/main/scala/abalone/StartingPosition.scala

```diff
@@ -20,6 +20,7 @@ case class StartingPosition(
   def initial = fen == format.Forsyth.initial
 }
 
+// @TODO: adapt these
```

Comment:
That's a long term 'nice to have' TODO. We dont need for initial release

Reply:
agreed